hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberStateFactory.java

```diff
@@ -292,6 +303,18 @@ private int handleMap(MemberStateImpl memberState, int count, Config config, Map
         return count;
     }
 
+    private int handleWan(MemberStateImpl memberState, int count, Map<String, LocalInstanceStats> wans,
+                          Set<String> longInstanceNames) {
+        for (Map.Entry<String, LocalInstanceStats> entry : wans.entrySet()) {
```

Comment:
I thought since this is not a data structure, it shouldn't really be restricted by `maxVisibleInstanceCount`. @emrahkocaman?